<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMusicWidget</class>
 <widget class="QWidget" name="QMusicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QMusicWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <widget class="QWidget" name="SubWidget" native="true">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>19</y>
          <width>233</width>
          <height>701</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>233</width>
          <height>701</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>211</width>
          <height>701</height>
         </size>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>30</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>üçäÊ©ôÂ≠êÈü≥‰πê</string>
         </property>
        </widget>
        <widget class="QWidget" name="widget" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>90</y>
           <width>211</width>
           <height>161</height>
          </rect>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>0</y>
            <width>191</width>
            <height>161</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="MusicLibPushButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Èü≥‰πêÈ¶Ü</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="autoRepeat">
              <bool>false</bool>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="VideoPushButton">
             <property name="text">
              <string>ËßÜÈ¢ë</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RadioPushButton">
             <property name="text">
              <string>ÁîµÂè∞</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>58</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Âú®Á∫øÈü≥‰πê</string>
         </property>
        </widget>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>280</y>
           <width>211</width>
           <height>241</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>280</y>
           <width>191</width>
           <height>241</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="MyfavoritePushButton">
            <property name="text">
             <string>ÊàëÂñúÊ¨¢</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="LocalMusicPushButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Êú¨Âú∞Ê≠åÊõ≤</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DownloadMusicPushButton">
            <property name="text">
             <string>‰∏ãËΩΩÊ≠åÊõ≤</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="HistoryPushButton">
            <property name="text">
             <string>Êí≠ÊîæÂéÜÂè≤</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PurchasePushButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Â∑≤Ë¥≠Èü≥‰πê</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>270</y>
           <width>58</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>ÊàëÁöÑÈü≥‰πê</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="MainWidget" native="true">
        <property name="geometry">
         <rect>
          <x>235</x>
          <y>19</y>
          <width>991</width>
          <height>701</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>991</width>
          <height>701</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>991</width>
          <height>701</height>
         </size>
        </property>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>991</width>
           <height>571</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>3</number>
         </property>
         <widget class="QWidget" name="MusicLib"/>
         <widget class="QWidget" name="Video">
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>270</y>
             <width>58</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>video</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="Radio">
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>410</y>
             <width>58</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="Myfavorite">
          <widget class="FavoriteWidget" name="favwidget" native="true">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>991</width>
             <height>571</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="LocalMusic">
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>250</y>
             <width>58</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>local</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="DownloadMusic">
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>290</y>
             <width>58</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="History">
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>320</y>
             <width>58</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="Purchase">
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>180</y>
             <width>58</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="ReSlider" name="playSlider">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>620</y>
           <width>991</width>
           <height>22</height>
          </rect>
         </property>
         <property name="pageStep">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="PlayButton">
         <property name="geometry">
          <rect>
           <x>468</x>
           <y>645</y>
           <width>35</width>
           <height>35</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="previousButton">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>654</y>
           <width>18</width>
           <height>18</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="NextButton">
         <property name="geometry">
          <rect>
           <x>523</x>
           <y>655</y>
           <width>18</width>
           <height>18</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="VolumeButton">
         <property name="geometry">
          <rect>
           <x>563</x>
           <y>655</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_13">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="lyircButton">
         <property name="geometry">
          <rect>
           <x>821</x>
           <y>655</y>
           <width>18</width>
           <height>18</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="lyircViewButton">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>640</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>ÂõæÁâá</string>
         </property>
        </widget>
       </widget>
       <widget class="QMusicAnimation" name="MusicViewAnimation" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>720</y>
          <width>1225</width>
          <height>16</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FavoriteWidget</class>
   <extends>QWidget</extends>
   <header location="global">favoritewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReSlider</class>
   <extends>QSlider</extends>
   <header location="global">reslider.h</header>
  </customwidget>
  <customwidget>
   <class>QMusicAnimation</class>
   <extends>QWidget</extends>
   <header location="global">qmusicanimation.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
